### DISCUSSION TITLE
pls give me a simple example about publish a message

### DISCUSSION THREAD

**Discussion by fuchanghai** [ag2ai/faststream#1972] [Q&A] [answered]
![image](https://github.com/user-attachments/assets/c60d1983-fa58-4def-8f2d-9beb2f28719c)

**Comment by deus-developer** [comment, marked as answer]
check this: https://faststream.airt.ai/latest/rabbit/publishing/#rabbit-publishing

**Comment by Lancetnik** [comment]
@fuchanghai the documentation link is correct - check it

Short summary: you should connect broker before publishing

```python
async with RabbitBroker() as broker:
     await  broker.publish(...)
```
